### ISSUE TITLE
Static routes and OSPF passive-interface

### ISSUE THREAD

**Issue by clxnetom** [bibigon812/bibigon812-quagga#23]
I am working on helping the networking team move the quagga configuration into Puppet. I am working on providing static routes and passive interfaces in OSPF (these are the two requirements I cannot seem to solve with the module out of the box).

I am getting there but I am pretty new at developing modules so while code reuse from other types/providers is a real help I am not sure if it is at as high quality as the rest of the module. This being said - is this something that you would like to merge into this module or have you decided not to have these features?

**Comment by bibigon812** [comment]
Did you mean this?

```
!
router ospf
 passive-interface eth0
 passive-interface eth1
```

```
ip route 1.1.1.1/1 1.1.1.1 250
```

**Comment by clxnetom** [comment]
Exactly!

Do you already have support for that?

**Comment by bibigon812** [comment]
No, I haven't. But I'll implement it soon.

**Comment by bibigon812** [comment]
Implemented in 3.4.0
